.gitignore

```diff
@@ -1,3 +1,5 @@
 node_modules
 .hubot_history
 .node-version
+.nyc_output/
+npm-debug.log
```

Comment:
the `--reporter=html` in the `npm test` script creates a `coverage` folder. I think it’s a good idea for local debugging of coverage, but we should add it to `.gitignore`